app/src/main/java/com/example/tasky/feature_agenda/domain/util/TaskScheduler.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.tasky.feature_agenda.domain.util
+
+import com.example.tasky.feature_agenda.domain.model.AgendaItem
+import com.example.tasky.feature_agenda.domain.model.Photo
+
+interface TaskScheduler {
+
+    fun scheduleEventTask(event: AgendaItem.Event, operation: OperationType, deletedPhotos: List<Photo> = emptyList())
```

Comment:
This name is a bit confusing, since it contains `Event` and `Task` (which are both agenda items), but I think one of these is a different context. I couldn't fully tell what this function does. Is it a task to schedule an event? Or is it the schedule event for a task?